internal/tuf/tuf.go

```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/secure-systems-lab/go-securesystemslib/cjson"
 )
 
+const currentMetadataVersion = 1
```

Comment:
let's start with 0, this should allow for "automatic" versioning of existing metadata, as the new field will have a default zero value of `0` due to being unset.